solidity-v1/dashboard/src/css/allocation-progress-bar.less

```diff
@@ -0,0 +1,31 @@
+.allocation-progress-bar {
+  token-allocation-data {
+  }
+
+  &__token-allocation {
+  }
```

Comment:
Same as above.

Reply:
Fixed in a21f6f12f41813bcb418acec4c3c15e685c7d940